Modules/Microsoft365DSC/DSCResources/MSFT_EXOSafeAttachmentPolicy/MSFT_EXOSafeAttachmentPolicy.psm1

```diff
@@ -254,27 +254,23 @@ function Set-TargetResource
         $StopProcessingPolicy = $false
         if ($Redirect -eq $true)
         {
+            if ($ActionOnError -eq $true){
```

Comment:
If the parameter is deprecated shouldn't we ignore it from all method and print a warning to the users letting them know that it is deprecated?

Reply:
Yes, but if this parameter is set in older configuration than you get an error. That's why I remove this parameter from the payload.